File: Modul-5/Soal-5/PRAK505-2410817210008-AhmadUlyani.c
#include <stdio.h>
void biodata(int tahunLahir, char nama[], char asal[]){
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahunLahir;
    printf("\n");
    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya Dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char nama[20], asal[15];
    scanf("%d",&tahunLahir);
    scanf("%s",&nama);
    scanf("%s",&asal);
    biodata(tahunLahir, nama, asal);
    return 0;
}

File: Modul-5/Soal-5/PRAK505-2410817210008-AhmadUlyani.py
def biodata(tahunlahir, nama, asal):
    tahunsekarang = 2020
    umur = tahunsekarang - tahunlahir
    print("")
    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahunsekarang}")
    print(f"Asal Saya Dari : {asal}")

def main():
    tahunlahir = int(input())
    nama = input()
    asal = input()

    biodata(tahunlahir, nama, asal)

if __name__ == "__main__":
    main()